pom.xml

```diff
@@ -634,7 +644,7 @@
               <argument>5m</argument>
               <argument>sh</argument>
               <argument>-c</argument>
-              <argument>until curl -sSk -o /dev/null http://${cryostat.itest.webHost}:${cryostat.itest.jfr-datasource.port}; do sleep 2; done</argument>
+              <argument>until podman ps --format '{{.Names}} {{.Status}}' | grep ${cryostat.itest.jfr-datasource.containerName} | grep healthy; do sleep 5; done</argument>
```

Comment:
Can we use this? 

```
[ -n "$(podman ps -a --filter health=healthy,name=abc --format={{.Names}})" ]
```

Reply:
I suppose, but I find that harder to read and it should do effectively the same thing, no?

Reply:
https://developers.redhat.com/blog/2019/04/18/monitoring-container-vitality-and-availability-with-podman#interacting_with_the_results_of_healthchecks

> When using a filter for health, you may use a value of healthy, starting, or unhealthy.

I just think that there are some fixed values we can specify in filters. As for grep, I am not sure if `podman` will ever decide to have a different display string. For example, `Healthy` for `health` or maybe localization.

Reply:
I suppose that's possible, but those strings are also defined in the libpod API as literals, and there are a lot of scripts out there built around these things, so I doubt podman will break compatibility by renaming the states. If it gets localized and that breaks it then we can just force the localization in the script, too. If capitalization is a concern (which it would be in your solution too?) then I can just do `grep -i` so it's case-insensitive.

Reply:
Yehh `grep -i` is the quickest way to go!